packages/api-nest/src/library/services/rss-feed-subscription.service.ts

```
@@ -0,0 +1,354 @@
+import { InjectQueue } from '@nestjs/bullmq'
+import { Queue } from 'bullmq'
+import { RssFeedMetadata } from '../../repositories/rss-feed.repository'
+import { RssFeedService } from '../../queue/services/rss-feed.service'
+import { QUEUE_NAMES, JOB_TYPES } from '../../queue/queue.constants'
+import { RssFeedEntity } from '../entities/rss-feed.entity'
+import { ContentType } from '../entities/library-item.entity'
+import { REPOSITORY_TOKENS } from '../../repositories/injection-tokens'
+import {
+  ILibraryItemRepository,
+  IRssFeedRepository,
+} from '../../repositories/interfaces'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)